src/components/ingestion/AbstractIngestionEngine.js

```diff
@@ -84,12 +92,14 @@ class AbstractIngestionEngine {
 		if (!(payload instanceof Payload)) {
 			done(new Error('ingestPayloadStream received non-Payload data'))
 		} else {
-			done(this.producer.send({
+			this.producer.send({
 				topic: dataTypes.STREAM.CONTAINER,
 				messages: [{
 					value: Payload.serialize(payload),
 				}],
-			}))
+			})
+				.then((result) => done(null, result))
```

Comment:
Super tiny thing, but I generally like to keep the first method on the same line as the Promise's closing paren, so this becomes
```
}).then()
  .catch()
```
But I'm not terribly wedded to that. Is this a thing we should care about, let be totally up to personal preference, or make circumstantial?

Reply:
Will do.

Reply:
Oh wait I'll bet this is because alignment of the `then/catch` methods is hard because we're using tabs instead of spaces and thus can't guarantee where they align. If that's the case and what's driving this then I retract my request!

Reply:
Ran in through prettier. The result both lints and looks great. Don't tell Dan.

Reply:
PRETTIERS?!?!?!?!?!?!